test/testcafe/framework/page-objects/BasePageObject.js

```diff
@@ -155,6 +155,23 @@ export default class BasePageObject {
     return this.getHelpLink().exists;
   }
 
+  getADPInstallRememdiationLink() {
+    return this.form.getLink('Install the Android Device Policy app on this device')
+  }
+
+  async adpOVInstallInstructionsMessageExists() {
+    return this.form.getByTextFn((_, element) =>
+      element.textContent === 'Go to Settings in Okta Verify and follow the instructions to install the Android Device Policy app').exists;
```

Comment:
It is because the text is broken up by a span element, so it actually looks like `Go to <span>Settings in Okta Verify</span> and follow the instructions. 

This is a workaround I found to ensuring we can assert on the entire text string instead of partials/substrings.